index.bs

```diff
@@ -6608,6 +6617,24 @@ At this time, one [=credential property=] is defined: the [=resident key credent
             If {{rk}} is not present, it is not known whether the credential is a [=discoverable credential=] or a [=server-side credential=].
 
             Note: some [=authenticators=] create [=discoverable credentials=] even when not requested by the [=client platform=]. Because of this, [=client platforms=] may be forced to omit the {{rk}} property because they lack the assurance to be able to set it to [FALSE]. [=[RPS]=] should assume that, if the `credProps` extension is supported, then [=client platforms=] will endeavour to populate the {{rk}} property. Therefore a missing {{rk}} indicates that the created credential is most likely a [=non-discoverable credential=].
+
+        :   <dfn>authenticatorDisplayName</dfn>
+        ::  This OPTIONAL property is a [=human palatability|human-palatable=] description of the credential's [=managing authenticator=],
+            chosen by the user.
+
+            The [=client=] MUST allow the user to choose this value,
+            MAY or MAY not present that choice during [=registration ceremonies=],
+            and MAY reuse the same value for multiple credentials with the same [=managing authenticator=] across multiple [=[RPS]=].
+
+            The [=client=] MAY query the [=authenticator=], by some unspecified mechanism, for this value.
+            The [=authenticator=] MAY allow the user to configure the response to such a query.
+            The [=authenticator=] vendor MAY provide a default response to such a query.
+            The [=client=] MAY consider a user-configured response chosen by the user,
+            and SHOULD allow the user to modify a vendor-provided default response.
+
+            If the [=[RP]=] includes an <code>[$credential record/authenticatorDisplayName$]</code> [=struct/item=] in [=credential records=],
+            the [=[RP]=] MAY offer this value, if present,
```

Comment:
Wait, this looks off:

> If the [=[RP]=] includes an <code>[$credential record/authenticatorDisplayName$]</code> [=struct/item=] in [=credential records=]

I think this should say "If the authenticator includes", right?

Reply:
No, this refers to a property of a _credential record_, which is a data structure internal to the RP.

But I do notice that it's ambiguous here whether "this value" refers to the credential record property or the extension output property. I'll try to disambiguate that.

Reply:
Oops, looks like this slipped through unaddressed. Fixed in PR #2005.